spec/clients/events_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+RSpec.describe Seam::Clients::Events do
+  let(:client) { Seam::Client.new(api_key: "some_api_key") }
+
+  describe "#list" do
+    let(:event_hash) { {event_id: "1234"} }
+
+    before do
+      stub_seam_request(:get, "/events/list?since=asd", {events: [event_hash]})
```

Comment:
actually, i think you can use the same helpers as line 27

```suggestion
      stub_seam_request(:get, "/events/list", {events: [event_hash]}).with(query: {since: "asd"})
```